### PR TITLE
support high-level RESTFUL API, listen on the same port as grpc

### PR DESCRIPTION
**Pull request by PowderLi**
Implement of #24746

### CODE REVIEW

**Comment by PowderLi** [comment]
rerun ut

File: pkg/util/paramtable/http_param.go [hunk 1]
```diff
@@ -23,4 +24,13 @@ func (p *httpConfig) init(base *BaseTable) {
 		Export:       true,
 	}
 	p.DebugMode.Init(base.mgr)
+
+	p.Port = ParamItem{
+		Key:          "proxy.http.port",
+		Version:      "2.1.0",
```
Comment (czs007) [inline pkg/util/paramtable/http_param.go:30]: better to be 2.3.0
you can change this in next pr.

**Pull request merged**